### DISCUSSION TITLE
Is there a comparison to testify?

### DISCUSSION THREAD

**Discussion by sheldonhull** [MarvinJWendt/testza#60] [Q&A]
I'm all for new testing tools!
I would like to see something mentioning testify in a general comparison to help understand why this might be a good choice over a more established project. 

I'm personally using the `is` which is very minimalist, and my default would be testify for more complicated tests. This seems really cool, just would like more understanding of what main benefits this has. 

thank you!

**Comment by MarvinJWendt** [comment]
Hi @sheldonhull, yes I think that would make sense! I think we could make a checklist which framework supports what. It might take some time tho, as we are currently working on new features for testza and the table would have to be updated very often 😄 

BTW: Your name looked familiar, and as it seems you starred [PTerm](https://github.com/pterm/pterm), which I also made. PTerm uses Testza for all tests. I switched from testify and the performance is pretty much the same, but with nicer output and less boilerplate in the test files! :)
